fiducialMarker/fiducialMarker.py

```diff
@@ -1,9 +1,10 @@
 from cv2 import aruco
 
 class FiducialMarker:
+    __dictionary = aruco.getPredefinedDictionary(aruco.DICT_4X4_50)

```

Comment:
Yeah, I think number of ID's is also restricted by the border and need to make markers recognizable from any orientation, but we should be good with 50 😄